test/kpod.bats

```diff
@@ -21,15 +21,15 @@ function teardown() {
 	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull debian:6.0.10
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} $KPOD_OPTIONS rmi debian:6.0.10
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi docker.io/debian:6.0.10
```

Comment:
These should still work without forcing everyone to prepend the registry?

Reply:
Yeah, just me being exact.